torch/nn/_functions/rnn.py

```diff
@@ -352,13 +361,16 @@ def forward(input, *fargs, **fkwargs):
 
         # Hack for the tracer that allows us to represent RNNs as single
         # nodes and export them to ONNX in this form
-        # It can be also used as a decorator at the higher level
         # Check the first argument explicitly to reduce the overhead of creating
-        # the lambda
+        # the lambda. We need special handling here because the forward()
+        # function gets reconstructed each and every time when RNN() is invoked
+        # and we don't want to pay the cost of decorator invocation
         import torch
         if torch._C._jit_is_tracing(input):
             import torch.onnx.symbolic
-            func = torch.onnx.symbolic.RNN_symbolic_builder(*args, **kwargs)(func)
+            decorator = torch.onnx.symbolic_override_first_arg_based(
```

Comment:
Is there any reason to use this decorator now? We know that the first arg has tracing enabled, so we could just always wrap `func` in `RNN_symbolic_builder`.

Reply:
I changed RNN_symbolic_builder to not apply symbolic_override any more. So we need to invoke it explicitly.